### DISCUSSION TITLE
Row select - Not able to select multiple rows

### DISCUSSION THREAD

**Discussion by vishwa3** [TanStack/table#2529] [General] [answered]
I am not able to select multiple rows -- If I select a row , already selected row gets deselected - that is I am able to select only 1 row at a time.

Sometimes I am able to select 2 rows one after other , but when I select 3rd row , one of the already selected rows gets deselected automatically , not allowing me to select all 3 rows one after other

Though the select all checkbox works fine , when checked all rows are selected .

I have supplied data consisting of 3 rows

Please help .

Codesandbox - https://codesandbox.io/s/upbeat-breeze-hyodx?file=/src/App.js

**Comment by vishwa3** [comment, marked as answer]
The issue is in index.js file ,
React-Table official codesandbox  - `ReactDOM.render(<App />, document.getElementById('root'))`

I was using React.StrictMode  as below -

```
const rootElement = document.getElementById("root");
ReactDOM.render(
  <React.StrictMode>
    <App />
  </React.StrictMode>,
  rootElement
);
```

On removing React.StrictMode , things worked , not sure though why Strict Mode was interfering

**Reply by tahv0** [reply]
This answer worked for me as well
